solidity/contracts/converter/BancorConverter.sol

```diff
@@ -138,25 +103,18 @@ contract BancorConverter is IBancorConverter, TokenHandler, SmartTokenController
       * @param  _token              smart token governed by the converter
       * @param  _registry           address of a contract registry contract
       * @param  _maxConversionFee   maximum conversion fee, represented in ppm
-      * @param  _reserveToken       optional, initial reserve, allows defining the first reserve at deployment time
-      * @param  _reserveWeight      optional, weight for the initial reserve
     */
     constructor(
         ISmartToken _token,
         IContractRegistry _registry,
-        uint32 _maxConversionFee,
-        IERC20Token _reserveToken,
-        uint32 _reserveWeight
+        uint32 _maxConversionFee
     )
         SmartTokenController(_token)
         ContractRegistryClient(_registry)
         public

```

Comment:
@yudilevi : Shouldn't that be `internal` now?

Reply:
Indeed - updated